src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {View} from 'react-native';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import useLocalize from '@hooks/useLocalize';
+import useSingleExecution from '@hooks/useSingleExecution';
+import Navigation from '@libs/Navigation/Navigation';
+import type {Route} from '@src/ROUTES';
+
+const advancedFilters = [
+    {
+        fieldName: 'type',
+        description: 'common.type' as const,
+        route: '/search/filters/type' as Route,
```

Comment:
Can we add the title here?
```suggestion
        route: '/search/filters/type' as Route,
        title: getFilterDisplayTitle(query, CONST.SEARCH.SYNTAX_ROOT_KEYS.TYPE),
```

Reply:
We'll probably need to move `advancedFilters` into the component if we do that, but I like keeping all the options in one place. We can also get rid of `fieldName` then.

Reply:
Sure I can change it if that is your preference, although that would not be mine.

The way I did is that all the static values (that will not change) can be defined outside of the component, and we don't have to wrap them with `useMemo`, whereas the dynamic values can be created when needed inside the `map`. I like it that way, but I guess I can put it inside and memoize.